blockstore/apps/api/v1/views/bundles.py

```diff
@@ -30,6 +32,10 @@ class Meta:
         fields = ('collection__uuid',)
 
 
+class HttpResponseSeeOtherRedirect(HttpResponseRedirectBase):
+    status_code = 303
```

Comment:
Today I learned when to use HTTP 303.  Nice! :+1:

Reply:
me too btw 😄 , @symbolist deserves all the kudos for the recommendation 👍